app/translations/cy/ccs_household_gb_eng_cy.po

```diff
@@ -49,7 +49,7 @@ msgid "A visitor is a person staying overnight in your household who usually liv
 msgstr "Mae ymwelydd yn golygu person sy’n aros dros nos sydd fel arfer yn byw mewn cyfeiriad arall."
 
 msgid "Remember to include partners and step-children as related."
-msgstr ""
+msgstr "Cofiwch gynnwys partneriaid a llysblant fel pobl sy'n perthyn."
```

Comment:
Dumb quotes alert. There are a few in these po files.

Reply:
While they should be correct in the translations, the schema build process fixes these ones.